prisma/schema.prisma

```diff
@@ -1,13 +1,13 @@
-generator client {
-  provider = "prisma-client-js"
-}
-
 datasource db {
   provider          = "postgresql"
   url               = env("DATABASE_URL")
   shadowDatabaseUrl = env("SHADOW_DATABASE_URL")
 }
 
+generator client {
```

Comment:
might want to consider putting this back at the top. It'll work either way, but by moving it back to the top of the file we can reduce the amount of changes in this PR :)